news/302.bug

```diff
@@ -0,0 +1,5 @@
+Errors while downloading sources
```

Comment:
It is better if the news file has only one line. This will be used to generate changelog.

Reply:
Fixed.